#
# Submitted-By: Robert Schwebel, 2006-03-05
# Committed-By: Robert Schwebel
#
# Error:
#
# arm-softfloat-linux-gnu-gcc  -g -O2 -Wall -g  -L/home/rsc/svn/ptxdist-0.10-trunk/projects/allyes-arm/local/arm-softfloat-linux-gnu/lib -L/home/rsc/svn/ptxdist-0.10-trunk/projects/allyes-arm/local/arm-softfloat-linux-gnu/usr/lib -Wl,-rpath-link -Wl,/home/rsc/svn/ptxdist-0.10-trunk/projects/allyes-arm/local/arm-softfloat-linux-gnu/usr/lib -o syslog-ng  main.o libsyslog-ng.a -lnsl   -L/home/rsc/svn/ptxdist-0.10-trunk/projects/allyes-arm/local/arm-softfloat-linux-gnu/usr/lib -Wl,-lglib-2.0 -L/home/rsc/svn/ptxdist-0.10-trunk/projects/allyes-arm/local/arm-softfloat-linux-gnu/usr/lib -Wl,-levtlog
# libsyslog-ng.a(cfg-lex.o): In function `yylex':
# /home/rsc/svn/ptxdist-0.10-trunk/projects/allyes-arm/build-target/syslog-ng-1.9.9/src/cfg-lex.c:1126: undefined reference to `yywrap'
# collect2: ld returned 1 exit status
# make[3]: *** [syslog-ng] Error 1
#
# Description:
#
# Fix found for example here: http://lists.gnu.org/archive/html/help-hurd/2004-12/msg00153.html
#
# State:
# 
# 2006-03-14	fixed in upstream trunk
# 2006-03-05	Submitted to Balázs Scheidler <bazsi@balabit.hu> 
#

diff -urN syslog-ng-1.9.9-orig/src/cfg-lex.l syslog-ng-1.9.9/src/cfg-lex.l
--- syslog-ng-1.9.9-orig/src/cfg-lex.l	2006-02-26 09:31:35.000000000 +0100
+++ syslog-ng-1.9.9/src/cfg-lex.l	2006-03-05 15:46:55.000000000 +0100
@@ -169,6 +169,7 @@
 %}
 
 %option nounput
+%option noyywrap
 
 white	[ \t]
 digit	[0-9]
